src/olympia/versions/models.py

```diff
@@ -223,7 +223,10 @@ def get_due_date_reason_q_objects(cls):
             # date even if the version has been disabled / not signed.
             'needs_human_review_from_cinder': Q(
                 needshumanreview__is_active=True,
-                needshumanreview__reason=NeedsHumanReview.REASONS.CINDER_ESCALATION,
+                needshumanreview__reason__in={
+                    NeedsHumanReview.REASONS.CINDER_ESCALATION,
+                    NeedsHumanReview.REASONS.CINDER_APPEAL_ESCALATION,
```

Comment:
There could be a follow-up to break these out - so they can be filtered separately - but this is fine for now.